pylint/reporters/text.py

```diff
@@ -35,8 +35,11 @@
 from pylint.reporters.ureports.text_writer import TextWriter
 
 if TYPE_CHECKING:
+    from pylint.lint import PyLinter
     from pylint.reporters.ureports.nodes import Section
 
+ColorMappingDict = Dict[str, Tuple[Optional[str], Optional[str]]]
```

Comment:
It might be possible to use a `NamedTuple` for the decorations (instead of `tuple[str | None, str | None]`).

Reply:
Not quite sure how to name the entries of the `NamedTuple` though. The first is obviously `colour`, but the second and third arguments would be something like `style1` and `style2`, right? Or did you have more specific names in mind?

Reply:
Maybe
```py
class Decoration(NamedTuple):
    color: str | None
    style: tuple[str, ...]
```